runhouse/rns/login.py

```diff
@@ -163,27 +149,44 @@ def login(
         )
         configs.set("default_autostop", int(new_autostop_mins))
 
-    if upload_config:
-        configs.load_defaults_from_file()
-        configs.upload_defaults(defaults=configs.defaults_cache)
+    local_config = configs.load_defaults_from_file()
```

Comment:
would local_config be empty for a first time login? we can either download from Den at the `rh_config_exists` check above or download it here if local_config is empty